highfidelity/hifi
# Case 21456, Case 21463: Clock On Tablet (81.0/master)

MS#[21463](https://highfidelity.manuscript.com/f/cases/21463/Philip-ask-Clock-on-Tablet)

# REQUIREMENTS
- A username of more than 14 characters, and a username between 9 and 14 characters are required.  Create them if you don't have them already.

# TEST PLAN
- Launch Interface in HMD mode
- Open tablet
- Clock should display on the top right with `Log in` underneath
- Log in with a username of over 14 characters
- Open tablet
- Truncated username appended with an ellipsis should be displayed underneath the clock
- Log out using `FILE > LOG OUT`
- Open tablet and click `Log in`
- Log in with a username of between 9 and 14 characters
- Open tablet
- The full username should be displayed below the clock in the tablet
